test/unit/routes/static.test.ts

```diff
@@ -0,0 +1,47 @@
+import { promises as fs } from "fs"
+import * as path from "path"
+import { tmpdir } from "../../utils/helpers"
+import * as httpserver from "../../utils/httpserver"
+import * as integration from "../../utils/integration"
+
+describe("static", () => {
+  let codeServer: httpserver.HttpServer | undefined
+  let testFile: string | undefined
+  let testFileContent: string | undefined
+
+  beforeAll(async () => {
+    const testDir = await tmpdir("static")
+    testFile = path.join(testDir, "test")
+    testFileContent = "static file contents"
+    await fs.writeFile(testFile, testFileContent)
+  })
+
+  afterEach(async () => {
+    if (codeServer) {
+      await codeServer.close()
+      codeServer = undefined
+    }
+  })
+
+  it("/static unauthed", async () => {
```

Comment:
nit: test should explain what the test is testing for ->
```suggestion
  it("should return a 200 if not authed and requesting files", async () => {
```